### ISSUE TITLE
When call a people who is in a call, there should give a notice

### ISSUE THREAD

**Issue by wjhtinger** [vector-im/riot-android#813]
Just like vector-web, give a notice that the callee is in a call already, and return immediately.

**Comment by ylecollen** [comment]
right, you should have an android toast. There is no alert dialog.
i tested it two weeks ago. It used to work.

**Comment by wjhtinger** [comment]
I tested again, the result is:
When the callee is a android client, there give a  toast, just show "The remote side failed to pick up". But I think the toast is some Blurred, give a exact reason is better.
When the callee is a web client, there will never a notice and the calling keep going on. Maybe this is the web client issue?
